msfdb

```diff
@@ -47,6 +47,8 @@ require 'msfenv'
 @components = %w(database webservice)
 @environments = %w(production development)
 
+ENV['PATH'] = "/usr/lib/postgresql/12/bin:#{ENV['PATH']}"
```

Comment:
I think this might need some more work, as this would only work for version 12 - but not 11, or 13 etc.
Folk may have their tooling installed elsewhere too, for instance this is the location on my mac:
```
$ locate pg_ctl
/usr/local/Cellar/postgresql@11/11.9/bin/pg_ctl
```

Reply:
Ah, well I never thought of that. I'll have to look at this and see what I can come up with.

Reply:
I think this problem/solution overlaps with this discussion where it's suggested not to use `pg_ctl` but `pg_ctlcluster` instead:
https://github.com/rapid7/metasploit-framework/issues/11369

For context, with a fresh install on a ubuntu machine `pg_ctl` is not on my path, but `pg_ctlcluster` is:
```
user@ubuntu:~$ pg_ctl
pg_ctl: command not found
user@ubuntu:~$ pg_ctlcluster
Error: Usage: /usr/bin/pg_ctlcluster <version> <cluster> <action> [-- <pg_ctl options>]
```

Note that later on that thread there's a link to the effort/semantic differences between the current tools we're using:
https://github.com/rapid7/metasploit-framework/issues/11369#issuecomment-468852471

Reply:
i.e. Adding `pg_ctl` onto your path works for the environment that you tested on, but it wouldn't always work for everyone